dev_guide/builds.adoc

```diff
@@ -511,6 +514,65 @@ customStrategy:
 ----
 ====
 
+[[pipeline-strategy-options]]
+
+== Pipeline Strategy Options
+
+The following options are specific to the
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline
+build strategy].
+
+[[jenkinsfile]]
+
+=== Providing the Jenkinsfile
+
+Pipeline builds require a Jenkinsfile which defines the steps that make
+up the pipeline that will be executed.  The Jenkinsfile can be provided
+in one of two ways.
+
+The first option is to include the Jenkinsfile definition directly in the build configuration,
```

Comment:
The way this currently reads, seems we would instead of a comma separating the above line with the below line, we want a period, where `define` would not be capitalized.